k8s-operator/apis/v1alpha1/types_proxyclass.go

```diff
@@ -36,10 +36,24 @@ type ProxyClassList struct {
 }
 
 type ProxyClassSpec struct {
+	// Configuration for tailscaled running in the proxy.
+	// +optional
+	TailscaledConfig *TailscaledConfig `json:"tailscaledConfig,omitempty"`
 	// Proxy's StatefulSet spec.
-	StatefulSet *StatefulSet `json:"statefulSet"`
+	// +optional
+	StatefulSet *StatefulSet `json:"statefulSet,omitempty"`
 }
 
+type TailscaledConfig struct {
+	// AcceptRoutes can be set to true to make the proxy to accept routes.
+	// from subnet routers and route traffic via exit nodes (defaults to false).
+	// https://tailscale.com/kb/1019/subnets
+	AcceptRoutes Bool `json:"acceptRoutes,omitempty"`
+}
+
+// +kubebuilder:validation:XValidation:rule="type(self) == string && (self=='true' || self=='false')",message="acceptRoutes must be set to a string value. Accepted values are 'true' and 'false'"
+type Bool string
```

Comment:
Defaults to this rather than being a bool so if we want in future we can swap `TailscaledConfig` defined here with https://github.com/tailscale/tailscale/blob/v1.60.0/ipn/conf.go#L14-L46 without breaking existing `ProxyClass`es.